023_agile_essential.storypoints.md

```diff
@@ -6,11 +6,11 @@ Improve your estimations and release planning.
 
 ## Description
 
-Story points are estimating the implementation complexity of a story, rather than the required time. Estimating in story points will improve the accuracy of your release plans, decrease the estimation effort for your teams and the variation in estimates due to external changes.
+Story points are estimating the implementation complexity of a story, rather than the required time. Estimating in Story Points will improve the accuracy of your release plans, decrease the estimation effort for your teams and the variation in estimates due to external changes.
 
-A widespread technic to estimate in story points is the **planning poker**. It is a format based on a shared understanding of a story. We at basysKom estimate with story points in the size of 1,2,3,5,8,13,20,40,100. Where 1 is a story with a clear idea and low implementation complexity and a 100 is a story which is super complex and not yet implementable. Stories that are takein into a sprint backlog usually end up in the range of 1 to 8 Story Points. 
+A widespread technique to estimate in story points is the *Planning Poker*. It is a format based on a shared understanding of a story. We at basysKom estimate with story points in the size of 1,2,3,5,8,13,20,40,100. Where 1 is a story with a clear idea and low implementation complexity and a 100 is a story which is super complex and not yet implementable. Stories that are taken into a sprint backlog usually end up in the range of 1 to 8 story points. 
 
-In a **planning poker** the product owner presents a story to the team, the team can now talk about the story. After that everyone estimates for them self. Once everyone has an estimation selected the team presents the own estimation to the each other. This is the point where the lowest estimation and the highest estimation need to explain why they decided upon the number. This way you can discover hidden complexity, find unspoken questions and increase the team understanding.
+In a *Planning Poker*-session the product owner presents a story to the team; the team can now talk about the story. After that, everyone estimates for themselves. Once everyone has an estimation selected the team presents the own estimation to the each other. This is the point where the lowest estimation and the highest estimation need to explain why they decided upon the number. This way you can discover hidden complexity, find unspoken questions and increase the team understanding.
```

Comment:
the bei "estimation to the each other" entfernen,
lowest estimation and the highest estimators (estimation => estimators)